10-gis.Rmd

```diff
@@ -23,16 +23,14 @@ library(gdalcubes)
 
 ```{r 09-gis-1-3, echo=FALSE, message=FALSE}
 # Only run code if qgis and relevant plugins are installed
-has_qgis_plugins = FALSE
-# Check to see that the qgisprocess package is installed:
-if ("qgisprocess" %in% installed.packages()) {
-  # Check that QGIS and required plugins are available to prevent errors in subsequent checks:
-  has_qgis_plugins = 
-    qgisprocess::has_qgis() && 
-    c("grassprovider", "processing_saga_nextgen") %in% 
-    qgisprocess::qgis_plugins()$name |>
-    all()
-}
+has_qgis_plugins = 
```

Comment:
My suggestion here: if `has_qgis()` is FALSE, there should not be any other `qgisprocess::*` functions called.

From a readability: it's worth creating separate objects stating if the environment has QGIS, grass, and saga, unless there are advantages of this more all-at-once-but-less-readable approach that I've not thought of.